packages/jazz-tools/src/coValues/interfaces.ts

```diff
@@ -191,45 +190,59 @@ export function subscribeToCoValue<V extends CoValue, Depth>(
   id: ID<V>,
   as: Account | AnonymousJazzAgent,
   depth: Depth & DepthsIn<V>,
-  listener: (value: DeeplyLoaded<V, Depth>) => void,
+  listener: (value: DeeplyLoaded<V, Depth>, unsubscribe: () => void) => void,
   onUnavailable?: () => void,
+  syncResolution?: boolean,
```

Comment:
Added the `syncResolution` option to use it to optimize the React and other frameworks subscriptions